services/common/rs_server_common/db/database.py

```diff
@@ -64,7 +64,7 @@ def open_session(self, url: str = ""):
         # make sure to initialize the session only once.
         with DatabaseSessionManager.lock:
             if (self._engine is None) or (self._sessionmaker is None):
-                self._engine = create_engine(url or self.url(), poolclass=NullPool)
+                self._engine = create_engine(url or self.url(), poolclass=NullPool, pool_pre_ping=True)
```

Comment:
Seen with Nicolas: without that the database sort of goes to sleep when unused and we get an error message when we "wake it up" for the first time